app/models/tasks/copy_image_task.rb

```diff
@@ -51,6 +47,5 @@ def update_image
     end
 
     image.save!
-    episode_or_podcast&.publish! if image.status_complete?
```

Comment:
This publishing-after-processing-completes has moved into the model `after_save` hooks.  For both images and media_resources.